tests/codeception/acceptance/100-BackendAdmin/BackendAdminCest.php

```diff
@@ -569,10 +569,8 @@ public function logoutAdminUserTest(\AcceptanceTester $I)
         $I->see('Dashboard');
         $I->click('Logout');
 
-        // Removed as we now unset the session cookie at logout
-        //$I->see('You have been logged out');
-
         $I->amOnPage('/bolt');
-        $I->see('Please log on');
+
+        $I->seeInCurrentUrl('/bolt/login');
```

Comment:
I don't think so....this is the only case that needed changing. I'm not sure I could reproduce the flash message running the test steps myself - so I changed it